### PR TITLE
implemented labeler which automatically adds RPC label to anything modifying RPC files

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
This was suggested by @thephez, I have not been able to test this at all.

Thoughts?

### CODE REVIEW

**Comment by PastaPastaPasta** [comment]
fixed

**Comment by thephez** [comment]
Hmm, just realized this would probably cut the number of messages @UdjinM6 gets from me by half. :joy: 
image

**Comment by PastaPastaPasta** [comment]
Hmm, maybe we shouldn't implement this then... It'd be a shame if you and udjin stopped talking so often

**Comment by PastaPastaPasta** [comment]
test failure unrelated, probably the deadlock we are trying to deal with

**Comment by UdjinM6** [comment]
Interesting idea but won't this cause too many false-positives? Changes like "fix typos", updates to internal interfaces, refactoring etc. should not be marked as RPC even when `rpc*` files are changed.

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Add 'RPC' to any changes within 'src/rpc/*'
+RPC:
+  - src/rpc/*
```
Comment (UdjinM6) [inline .github/labeler.yml:3]: some rpcs are in `src/wallet/rpc*`

**Comment by thephez** [comment]
> Interesting idea but won't this cause too many false-positives? Changes like "fix typos", updates to internal interfaces, refactoring etc. should not be marked as RPC even when `rpc*` files are changed.

True. Personally I think I would prefer false positives over false negatives. I guess an option would be to add a new RPC "triage" label automatically to flag them likely PRs. But then it would still require manual intervention to assign the proper label (or remove in case of false positive).

Maybe not as helpful an idea as I thought it would be :thinking:

**Pull request merged**